website/pages/index.js

```diff
@@ -61,6 +62,15 @@ function Home() {
   const context = useContext(DocusaurusContext);
   const {siteConfig = {}} = context;
 
+  const feedbackUrl = `${siteConfig.baseUrl}feedback/`;
```

Comment:
We need a API (wrapper function) for these and not let people concatenating the URLs themselves.

Reply:
Let's add it on hack md. And think of the wrapper function name.

This PR is already quite big